Menus/settings_menu.gd

```diff
@@ -65,6 +81,18 @@ func _on_quit_button_pressed():
 	# todo - save before quitting if in-game
 	get_tree().quit()
 
+func _on_window_mode_option_selected(_value):
```

Comment:
So if we move these utilities to their own class, these calls would become something like Globals.window_settings.set_window_mode, etc. Otherwise, at a glance, the implementation in this class looks solid to me.

Reply:
Done